include/bluetooth/gap.h

```diff
@@ -128,6 +128,13 @@ enum {
  */
 #define BT_GAP_ADV_HIGH_DUTY_CYCLE_MAX_TIMEOUT  128
 
+#define BT_GAP_DATA_LEN_DEFAULT                 0x001b /* 27 bytes */
```

Comment:
Should we rather use a `MIN` than `DEFAULT`? In many cases, using `MAX` is the preferred option to reduce power consumption and latency.